Update backup-azure-vms-encryption.md

Ramyaharinarthini opened the pull request:
The below line has to be added in the document as there is an option to swap the os disk of the VM with the restored disk without recreating it again.

 Note: After restoring the VM disk swap the OS Disk of the Original VM with the restored VM disk without having to recreate it again learn more: https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/

Ramyaharinarthini committed:
Update backup-azure-vms-encryption.md


The below line has to be added in the document as there is an option to swap the os disk of the VM with the restored disk without recreating it again.

 Note: After restoring the VM disk swap the OS Disk of the Original VM with the restored VM disk without having to recreate it again learn more: https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/

AbhishekMallick-MS commented on articles/backup/backup-azure-vms-encryption.md:
```diff
@@ -153,6 +153,7 @@ Encrypted VMs can only be restored by restoring the VM disk as explained below.
 Restore encrypted VMs as follows:
 
 1. [Restore the VM disk](backup-azure-arm-restore-vms.md#restore-disks).
+  + Note: After restoring the VM disk swap the OS Disk of the Original VM with the restored VM disk without having to recreate it again learn more: https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/ +
```
Please use the following format with correct indentation. Before you merge, please send the updated section in WORD for review (attach the file).

>[!NOTE]
>After restoring the VM disk, swap the OS Disk of the original VM with the restored VM disk without recreating it. [Learn more](https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/).

AbhishekMallick-MS reviewed (changes requested):
@Ramyaharinarthini 

Please make the change and send for review in word.

Ramyaharinarthini committed:
Update backup-azure-vms-encryption.md

Updated line 156 as below
>[!NOTE]
>After restoring the VM disk, swap the OS Disk of the original VM with the restored VM disk without recreating it. [Learn more](https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/).

Ramyaharinarthini committed:
Update backup-azure-vms-encryption.md

Updated line 156 as below
>[!NOTE]
>After restoring the VM disk, swap the OS Disk of the original VM with the restored VM disk without recreating it. [Learn more](https://azure.microsoft.com/en-in/blog/os-disk-swap-managed-disks/).

Ramyaharinarthini committed:
Update backup-azure-vms-encryption.md

updated the Note line 156

AbhishekMallick-MS reviewed (approved):
Looking good. Signing off.

ktoliver committed:
PR review edit
